### PR TITLE
Feat: Added About Us and Privacy Policy screens

### LINKED ISSUE #1064: Add About Us screen and privacy policy
**Summary:** 

Add About Us screen and privacy policy

### PR DESCRIPTION
**Pull request by tarun0**
Fixes #1064 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -30,6 +34,7 @@
     <string name="locale">Locale</string>
     <string name="confirm_delete_client_image">Are you sure you want to delete this client\'s image?</string>
     <string name="default_instance_url" translatable="false">demo.openmf.org</string>
+    <string name="url_privacy_policy" translatable="false">http://www.google.com</string>
```
Comment (tarun0) [inline mifosng-android/src/main/res/values/strings.xml]: @therajanmaurya Please make sure to update it. I added this as demo server was down when I tested.

**Comment by tarun0** [comment]
d1

**Comment by tarun0** [comment]
Also, the build is passing locally. Will need to check what it needs regarding the DBFlow dependency.

File: mifosng-android/src/main/res/values/strings.xml (hunk 1 above)
Comment (therajanmaurya) [reply mifosng-android/src/main/res/values/strings.xml]: Ok

**Comment by therajanmaurya** [comment]
Please add License feature too.

**Comment by tarun0** [comment]
> Please add License feature too.

Added

**Comment by droidchef** [comment]
CI is failing, it needs to be resolved before we can take a look at this PR. Please do that and re-create a PR. Thank you.

**Pull request closed without merging**